package.json

```diff
@@ -113,11 +113,18 @@
   },
   "dependencies": {
     "@inlet/react-pixi": "^0.5.7",
+    "@types/d3": "^5.7.2",
+    "@types/d3-contour": "^1.3.0",
+    "@types/react-leaflet": "^2.2.1",
     "@types/recharts": "^1.1.16",
     "color": "^3.1.0",
+    "d3": "^5.9.7",
```

Comment:
As you mentioned, it seems odd to include all of D3 in order to do the contour work.  

OTOH, it looks great. 
Also OTOH: Now that we are including D3 maybe we can lean on it in other views.

Are you interested in trying to implement the contours without D3?